wdl/transforms/biscayne/src/test/scala/wdl/transforms/biscayne/linking/expression/values/BiscayneValueEvaluatorSpec.scala

```diff
@@ -217,4 +217,128 @@ class BiscayneValueEvaluatorSpec extends AnyFlatSpec with CromwellTimeoutSpec wi
       e.evaluateValue(Map.empty, NoIoFunctionSet, None) shouldBeValid EvaluatedValue(expectedArray, Seq.empty)
     }
   }
+
```

Comment:
Thorough tests!

Reply:
As discussed in mob time, to make them even more thorough... we should test with every primitive data type.

https://github.com/openwdl/wdl/blob/wdl-1.1/SPEC.md#primitive-types